support-frontend/app/controllers/PaperSubscriptionForm.scala

```diff
@@ -40,21 +40,7 @@ class PaperSubscriptionForm(
   implicit val a: AssetsResolver = assets
 
 
-  def displayForm(): Action[AnyContent] =
```

Comment:
We don't need two separate urls for regular and guest checkout any more so this removes the one called `displayGuestForm` and changes `displayForm` to show the guest checkout.